src/sage/tensor/modules/finite_rank_free_module.py

```diff
@@ -545,9 +545,16 @@ class :class:`~sage.modules.free_module.FreeModule_generic`
 from sage.sets.family import Family, TrivialFamily
 from sage.structure.parent import Parent
 from sage.structure.unique_representation import UniqueRepresentation
+from sage.tensor.modules.comp import Components
```

Comment:
I think this should go into an `if TYPE_CHECKING`

Reply:
Could, but doesn't have to.